contracts/SmartSession.sol

```diff
@@ -251,21 +253,26 @@ contract SmartSession is ISmartSession, SmartSessionBase, SmartSessionERC7739 {
         /*.•°:°.´+˚.*°.˚:*.´•*.+°.•°:´*.´•*.•°.•°:°.´:•˚°.*°.˚:*.´+°.•*/
         // Check UserOp policies
         // This reverts if policies are violated
-        vd = $userOpPolicies.check({
+        uint256 p;
+        (vd, p) = $userOpPolicies.check({
```

Comment:
would be easier to read with proper names (eg validationData and policies)

Reply:
https://github.com/erc7579/smartsessions/pull/85/commits/69ea13eb5fff6b5f3cd1ff3329b6fb2140dec149